bench/query.go

```diff
@@ -132,6 +132,90 @@ func (q *QueryGenerator) Random(maxN, depth, maxargs int, idmin, idmax uint64) *
 	}
 }
 
+// RandomRangeQuery returns a randomly generated either sum/range query.
```

Comment:
Change comment to "RandomRangeQuery returns a randomly generated sum or range query."